library/db/src/main/java/org/keynote/godtools/android/db/Contract.java

```diff
@@ -47,23 +47,28 @@
 
         public static final String COLUMN_CODE = "code";
         public static final String COLUMN_ADDED = "added";
+        public static final String COLUMN_NAME = "name";
 
         public static final Field FIELD_ID = TABLE.field(COLUMN_ID);
         public static final Field FIELD_CODE = TABLE.field(COLUMN_CODE);
         public static final Field FIELD_ADDED = TABLE.field(COLUMN_ADDED);
+        public static final Field FIELD_NAME = TABLE.field(COLUMN_NAME);
 
-        static final String[] PROJECTION_ALL = {COLUMN_ID, COLUMN_CODE, COLUMN_ADDED};
+        static final String[] PROJECTION_ALL = {COLUMN_ID, COLUMN_CODE, COLUMN_ADDED, COLUMN_NAME};
 
         private static final String SQL_COLUMN_CODE = COLUMN_CODE + " TEXT NOT NULL";
         private static final String SQL_COLUMN_ADDED = COLUMN_ADDED + " INTEGER";
+        private static final String SQL_COLUMN_NAME = COLUMN_NAME + " TEXT";
         private static final String SQL_PRIMARY_KEY = uniqueIndex(COLUMN_CODE);
 
-        static final Expression SQL_WHERE_PRIMARY_KEY = FIELD_CODE.eq(bind());
+        static final Expression SQL_WHERE_PRIMARY_KEY = FIELD_CODE.eq(bind()).and(FIELD_NAME.eq(bind()));
```

Comment:
the name should not be part of the primary key. this where clause is for looking up languages by the primary key